### PR TITLE
fix(api): Update v2 API Definitions 

### PR DESCRIPTION
**Pull request by jeremyjaouen**
## Description

Correction of documentation and missings routes

**Fixes** # 5788

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x
- [x] 20.10.x (master)

## Checklist

- [ ] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: doc/API/centreon-api-v2.yaml [hunk 1]
```diff
@@ -2784,9 +3114,10 @@ components:
       properties:
         is_forced:
           type: boolean
-          required: false
           description: "Indicates whether the check is forced (do not take into account the check timeperiod)"
           example: true
+      required:
+        - is_forced
```
Comment (kduret) [inline doc/API/centreon-api-v2.yaml:3120]: is_forced is not required

File: doc/API/centreon-api-v2.yaml [hunk 2]
```diff
@@ -1728,7 +2044,7 @@ paths:
                         data:
                           type: array
                           items:
-                            type: float
+                            type: number
```
Comment (kduret) [inline doc/API/centreon-api-v2.yaml:2047]:
```suggestion
                            type: number
                            format: float
```

File: doc/API/centreon-api-v2.yaml [hunk 3]
```diff
@@ -1743,11 +2059,11 @@ paths:
                               description: label to display in legend
                               example: 'Last:1.00'
                         minimum_value:
-                          type: float
+                          type: number
```
Comment (kduret) [inline doc/API/centreon-api-v2.yaml:2062]:
```suggestion
                          type: number
                          format: float
```

File: doc/API/centreon-api-v2.yaml [hunk 4]
```diff
@@ -1743,11 +2059,11 @@ paths:
...
                           description: minimum value
                           example: 1.0
                         maximum_value:
-                          type: float
+                          type: number
```
Comment (kduret) [inline doc/API/centreon-api-v2.yaml:2066]:
```suggestion
                          type: number
                          format: float
```

File: doc/API/centreon-api-v2.yaml [hunk 5]
```diff
@@ -3162,7 +3499,7 @@ components:
               description: "Is the host flapping or not"
               example: false
             percent_state_change:
-              type: float
+              type: number
```
Comment (kduret) [inline doc/API/centreon-api-v2.yaml:3502]:
```suggestion
              type: number
              format: float
```

File: doc/API/centreon-api-v2.yaml (hunk 2 above)
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:2047]:
```suggestion
                            type: number
                            format: float
```

File: doc/API/centreon-api-v2.yaml (hunk 3 above)
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:2062]:
```suggestion
                          type: number
                          format: float
```

File: doc/API/centreon-api-v2.yaml (hunk 4 above)
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:2066]:
```suggestion
                          type: number
                          format: float
```

File: doc/API/centreon-api-v2.yaml [hunk 6]
```diff
@@ -1791,52 +2121,52 @@ paths:
                       type: object
                       properties:
                         start:
-                          type: integer
+                          type: string
```
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:2124]: why should it be a string ?

File: doc/API/centreon-api-v2.yaml [hunk 7]
```diff
@@ -2784,9 +3114,10 @@ components:
       properties:
         is_forced:
           type: boolean
-          required: false
           description: "Indicates whether the check is forced (do not take into account the check timeperiod)"
           example: true
+      required:
```
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:3119]: This is not required anymore : https://github.com/centreon/centreon/pull/8854/files#diff-e32a83c866e6a6b0e3ea466fc26db9a1L2787

File: doc/API/centreon-api-v2.yaml (hunk 1 above)
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:3120]:
```suggestion
```

File: doc/API/centreon-api-v2.yaml (hunk 5 above)
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:3502]:
```suggestion
              type: number
              format: float
```

File: tests/api/features/Authentication.feature [hunk 8]
```diff
@@ -0,0 +1,13 @@
+Feature:
+    In order to access the API
+    As a visitor
+    I want to login to the API
+
+    Background:
+        Given a running instance of Centreon Web API
+
+    Scenario:
```
Comment (sc979) [inline tests/api/features/Authentication.feature:9]: Kindly, could you add the succesful connection test too ?

Comment (jeremyjaouen) [reply tests/api/features/Authentication.feature:9]: I discuss about this with @kduret, succesful connection test are already handle into the ApiContext : Given I am logged in

**Pull request merged**